index.html

```diff
@@ -19,12 +19,13 @@
 </head>
 <body>
   <header>
-    <div class="container header-container">
+    <div class="container header-container row align-baseline">
```

Comment:
No one element should be a container and a row or a container and a column, those are separate tools. You should probably have a `row` as a child to this `container`.